lib/timeline/component/item/Item.js

```diff
@@ -181,6 +182,80 @@ Item.prototype._repaintDeleteButton = function (anchor) {
 };
 
 /**
+ * Repaint a onChange tooltip on the top right of the item when the item is selected
+ * @param {HTMLElement} anchor
+ * @protected
+ */
+Item.prototype._repaintOnItemUpdateTimeTooltip = function (anchor) {
+  if (!this.options.tooltipOnItemUpdateTime) return
```

Comment:
Please add a semicolon at the end of the line. Without it the code is more error-prone.